lib/common/version.go

```diff
@@ -0,0 +1,6 @@
+package common
+
+const (
+	TransactionMessage_Version1 = "tx_version1"
```

Comment:
Please remove `_`, `TransactionMessage_Version1` -> `TransactionMessageVersion1`. How about remove `Message` in name? Just `TransactionVersion`, `Version1` does not mean anything, I think.